QSB/Instruments/QSBCamera/CameraController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using UnityEngine;
+
+namespace QSB.Instruments.QSBCamera
+{
+    class CameraController : MonoBehaviour
```

Comment:
is this taken from FreeCam, or is it something else? :D

Reply:
the input stuff and rotation is taken from playercameracontroller and tweaked slightly, the position stuff is custom